src/SFA.DAS.CandidateAccount.Application/Application/Commands/UpsertApplication/UpsertApplicationCommandHandler.cs

```diff
@@ -60,7 +60,11 @@ public async Task<UpsertApplicationCommandResponse> Handle(UpsertApplicationComm
         });
 
         await UpsertAdditionalQuestions(command, cancellationToken, application.Item1);
-        await UpsertEmploymentLocations(command, application.Item1, cancellationToken);
+        if (command.EmploymentLocation != null)
+        {
+            await UpsertEmploymentLocations(command, application.Item1, cancellationToken);
+        }
+        
         if (application.Item2)
```

Comment:
We need to set below here the section status to `NotRequired` for employer location

Reply:
✨ Update EmploymentLocationStatus assignment logic

- Modify `EmploymentLocationStatus` to use a conditional check.
- Assign status based on the presence of `command.EmploymentLocation`.
- If not null, use `(short)command.IsEmploymentLocationComplete`.
- If null, assign `(short)SectionStatus.NotRequired`.
- Enhance handling of employment location status in the command.